examples/service-clients/azure-client/external-controller/src/view.ts

```diff
@@ -76,15 +80,17 @@ function makeAudienceView(audience?: IAzureAudience): HTMLDivElement {
 	onAudienceChanged();
 	audience.on("membersChanged", onAudienceChanged);
 
-	wrapperDiv.appendChild(audienceDiv);
+	wrapperDiv.append(audienceDiv);
 	return wrapperDiv;
 }
 
 export function makeAppView(
 	diceRollerControllers: IDiceRollerController[],
 	audience?: IAzureAudience,
 ): HTMLDivElement {
-	const diceRollerViews = diceRollerControllers.map(makeDiceRollerView);
+	const diceRollerViews = diceRollerControllers.map((controller) =>
```

Comment:
Is this due to unbound?  If so, probably just prefer to change makeDiceRollerView to be an arrow function rather than adding the extra anonymous function here.

Reply:
Yes, that was the motivation. Can do.

Reply:
No, excuse me, the rule is this one here: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v48.0.1/docs/rules/no-array-callback-reference.md

I had forgotten about this, but the rationale here is sound. I think this is probably an important rule for us to enforce in production code but may or may not make sense to enforce in examples.

If we would prefer to disable this rule for our examples, I can add an exception to the local config (rather than adding an inline disable).

Reply:
Interesting, I hadn't seen that unicorn rule before.  Either way is fine with me, your choice.